Ports/pt2-clone/patches/PATH_MAX.patch

```diff
@@ -0,0 +1,78 @@
+Common subdirectories: pt2-clone-1.28/src/gfx and pt2-clone-1.28d/src/gfx
```

Comment:
Never seen this in a patch before, what does it mean?

Reply:
i just used diff -u so i have a one big patch with all changed files and it seems like it left this there